### PR TITLE
Add PHPUnit

### PR DESCRIPTION
**Pull request by mustorze**
Hi!

I've added Phpunit and Orchestra to helps with tests.

hope your enjoy.

have nice tests xD

### CODE REVIEW

File: tests/Functional/initializeTest.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace Dpc\GuzzleClient\Tests\Functional;
+
+use Dpc\GuzzleClient\GuzzleClient;
+use Dpc\GuzzleClient\Tests\TestBase;
+
+class initializeTest extends TestBase
+{
+    /** @test **/
+	function guzzle_initialize()
```
Comment (Dylan-DPC-zz) [inline tests/Functional/initializeTest.php]: I don't need this test

**Review by Dylan-DPC-zz — CHANGES_REQUESTED** [review]

**Comment by Dylan-DPC-zz** [comment]
references #16

**Comment by mustorze** [comment]
Okay, test deleted.

If your have some function do you like to test, i can do it. I really don't know anything about your package, then i no have idea to do the tests

File: composer.json [hunk 2]
```diff
@@ -6,7 +6,10 @@
     "require": {
         "php": ">=7.1",
         "guzzlehttp/guzzle": "^6.3",
-        "illuminate/support": "5.4.*||5.5.*"
+        "illuminate/support": "5.4.*||5.5.*",
+        "phpunit/phpunit": "^6.3",
+        "orchestra/database": "^3.5",
```
Comment (Dylan-DPC-zz) [inline composer.json]: do i really need database? cause the package isn't interacting with DB at all.

**Review by Dylan-DPC-zz — CHANGES_REQUESTED** [review]

**Comment by Dylan-DPC-zz** [comment]
Thx. But someone else is already attempting the tests.

**Comment by mustorze** [comment]
yea, don't needed. 

i put by habit haha.

--

Ok, if in the future had some tests to do, i will be happy helping the package. =)

**Comment by Dylan-DPC-zz** [comment]
If you want you can write tests for the function. Haven't heard from the other guy as yet

**Pull request merged**